apps/src/javalab/Theater.js

```diff
@@ -4,23 +4,18 @@ export default class Theater {
     this.context = null;
   }
 
-  afterInject() {
-    this.canvas = document.getElementById('theater');
-    this.context = this.canvas.getContext('2d');
-    if (this.context) {
-      // Fill the canvas with a white "default" background
-      this.context.fillStyle = 'white';
-      this.context.rect(0, 0, 400, 400);
-      this.context.fill();
-    }
-  }
-
   handleSignal(data) {
     var imageString = 'data:image/gif;base64,' + data.detail.image;
-    var base_image = new Image();
-    base_image.src = imageString;
-    base_image.onload = () => {
-      this.context.drawImage(base_image, 0, 0);
-    };
+    var imgElement = this.getImgElement();
```

Comment:
nit / question - do we have a preference for `let`/`const` vs `var` here?

Reply:
good question, we do prefer `let`/`const`, I'll update this